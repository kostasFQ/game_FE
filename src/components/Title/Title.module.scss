.title {
  &__container {
    display: flex;
    justify-content: center;
    align-items: flex-end;
    width: 100%;
    height: 50px;

    &__text {
      font-family: 'Otribon', sans-serif;
      font-size: 2em;
      font-weight: bolder;
      text-transform: uppercase;
      background: linear-gradient( rgba(2, 2, 2, 0.473) 10%, rgba(250, 40, 40, 0.363));
      background-clip: text;
      -webkit-background-clip: text;
      -webkit-text-fill-color: transparent;
      color: #0B2349;
      display: table;
      word-wrap: none;
    }
  }
}