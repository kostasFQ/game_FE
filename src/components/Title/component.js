import React from 'react';
import styles from './Title.module.scss';

const Title = () => (
  <div className={styles.title__container}>
  <div className={styles.title__container__text}>
    clicatron3000
  </div>
  </div>
)

export default Title;