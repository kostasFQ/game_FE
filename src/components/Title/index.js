import Title from './TitleComponent';

export default Title;