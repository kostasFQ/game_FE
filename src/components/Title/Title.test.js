import React from 'react';
import Title from './component';
import renderer from 'react-test-renderer';

test('Button should be render', () => {
  const instance = renderer.create(
    <Title />
  );
  let component = instance.toJSON();
  expect(component).toMatchSnapshot();
})