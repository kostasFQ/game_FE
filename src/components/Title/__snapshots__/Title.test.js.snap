// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should be render 1`] = `
<div
  className="title__container"
>
  <div
    className="title__container__text"
  >
    clickatron3000
  </div>
</div>
`;
